src/main/java/TaskList.java

```diff
@@ -0,0 +1,116 @@
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * TaskList reads and do all the instructions, and returns them as Task
+ */
+
+public class TaskList {
+
+    String description;
+    List<String> allInstructions = new ArrayList<>();
+    static List<Task> store = new ArrayList<>();
+    Ui ui = new Ui();
```

Comment:
Better to just pass ui as an argument to the methods you want to use since you already have a ui object in parser